cmd/govim/testdata/scenario_default/quickfix_from_other.txt

```diff
@@ -0,0 +1,95 @@
+# Test that the quickfix window isn't updated by govim if it isn't populated by govim
+
+vim ex 'e main.go'
+vim ex 'call setqflist([{\"filename\":\"foo\"}, {\"filename\":\"bar\"}, {\"filename\":\"baz\"}], \"r\")'
```

Comment:
This is a great test.

One thing I think we need to do before calling this is verify that the quickfix list gets populated with the error diagnostics from `gopls`. That's not essential to this test, but instead a more realistic reflection of real life. So that way you end up with a call to `vimexprwait` before this call, then as you have it another one afterwards.

Reply:
@myitcv OK, I added an assertion for the first error 35c475d